src/resolvers/subjectResolvers.ts

```diff
@@ -19,12 +19,12 @@ export const Query = {
   ): Promise<GQLSubject> {
     const list = await fetchSubjects(context);
     return list
-      .filter(s => s.metadata.visible === true)
+      .filter(s => (s.metadata && s.metadata.visible) || true)
```

Comment:
Vil ikke dette filteret returnere true for alle cases?
Om du ønsker å returnere hvor visible er true eller ikke satt så er det vel mulig å sjekke med `(s.metadata && s.metadata.visible !== false)`
Evt. `(s.metadata && s.metadata.visible) || s.metadata && s.metadata.visible === undefined)`